proposals/simd/ImplementationStatus.md

```diff
@@ -144,30 +144,32 @@
 | `f64x2.convert_i64x2_s`    | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
 | `f64x2.convert_i64x2_u`    | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
 | `v8x16.swizzle`            |                           |                    | :heavy_check_mark: |                    |
-| `v8x16.shuffle`            |                           |                    | :heavy_check_mark: | :heavy_check_mark: |
-| `i16x8.load8x8_s`          |                           |                    |                    |                    |
-| `i16x8.load8x8_u`          |                           |                    |                    |                    |
-| `i32x4.load16x4_s`         |                           |                    |                    |                    |
-| `i32x4.load16x4_u`         |                           |                    |                    |                    |
-| `i64x2.load32x2_s`         |                           |                    |                    |                    |
-| `i64x2.load32x2_u`         |                           |                    |                    |                    |
-| `i8x16.narrow_i16x8_s`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i8x16.narrow_i16x8_u`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.narrow_i32x4_s`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.narrow_i32x4_u`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.widen_low_i8x16_s`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.widen_high_i8x16_s` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.widen_low_i8x16_u`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i16x8.widen_high_i8x16_u` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i32x4.widen_low_i16x8_s`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i32x4.widen_high_i16x8_s` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i32x4.widen_low_i16x8_u`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
-| `i32x4.widen_high_i16x8_u` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `v8x16.shuffle`            |            `-msimd128`[5] | :white_check_mark:[5] | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
Nit: need to realign all the | ?

Reply:
I was trying to keep the diff manageable, but I guess if it's in a separate commit that's ok.

Reply:
Sg. The diff view should be able to hide whitespace to make things more manageable https://github.blog/2018-05-01-ignore-white-space-in-code-review/